infra/cdk/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "cdk",
+  "version": "0.1.0",
+  "bin": {
+    "cdk": "bin/cdk.js"
+  },
+  "scripts": {
+    "build": "tsc",
+    "watch": "tsc -w",
+    "test": "jest",
+    "cdk": "cdk",
+    "deploy":"cdk deploy --profile roadoxe"
```

Comment:
was adding a profile in the command a good idea? Because we might name our profiles differently

Reply:
I guess yeah it should be just cdk deploy